### DISCUSSION TITLE
document.getElementById()

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#8740] [Help]
Hello everyone.
why document.getElementById() does not work with embed component
@melohagan 

![d](https://user-images.githubusercontent.com/106697208/202905214-b6c68967-a898-4cfb-8cfb-645fa75c8509.PNG)

**Comment by andz-bb** [comment]
hey @rafikApp, check out the solution to #8095, there's a working example of it in the solution there
